### ISSUE TITLE
Resource Override doesn't work on Chrome 84

### ISSUE THREAD

**Issue by laukstein** [kylepaulsen/ResourceOverride#38]
On Chrome 84 Network and Sources, etc. shows Resource Override URL to URL doesn't have any affect, the extension is n o longer working.

**Comment by kylepaulsen** [comment]
Could you give more details please? I wasn't able to reproduce. I was using Chrome Version 84.0.4147.105 (Official Build) (64-bit) on Mac OSX 10.13.6 and was able to redirect one website to another.

**Comment by laukstein** [comment]
I reproduced the issue on latest Chrome Windows 10. Used valid regex for URL to URL override.

**Comment by kylepaulsen** [comment]
Valid regex? The url match strings are not run through a regex engine. The "*" syntax is custom. Are you able to post the url strings?

Also just tested on Chrome Version 84.0.4147.105 (Official Build) (64-bit) on windows 10.

**Comment by AgDude** [comment]
I am running into this on chrome 85.0.4183.102.  It is a bit more nuanced that the initial report.

I have the following override: 
`https://static.mycdn.com/*/my-app.min.js` --> `http://127.0.0.1:8081/my-app.js`

It loads the non-minified version from localhost, as expected. This can be seen by the page content and the network tab.  The minified version has a 307 (internal redirect) in the network tab.

However, when I go to the sources tab in dev tools I cannot open the non-minified version. The only file available in the sources tab is `https://static.mycdn.com/latest/my-app.min.js`. 

So in summary the plugin is still overriding the page content, but not the dev tools.
